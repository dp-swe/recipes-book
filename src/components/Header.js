import React from 'react';
import styles from './styles/header.module.scss';
import texts from './texts';

const Header = () => (
    <header className={styles.header}>
        <div className={styles.header__title}>{texts.TITLE_HEADER}</div>
    </header>
)

export default Header;