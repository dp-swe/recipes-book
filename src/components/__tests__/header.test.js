import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Header from '../Header';

describe('Header Component', () => {

    it('should render without broken', () => {
        const wrapper = shallow(<Header/>);
        expect(wrapper.exists()).toBe(true);
    })

    it('should render correctly', () => {
        const tree = renderer.create(<Header/>).toJSON();
        expect(tree).toMatchSnapshot();
    })
})