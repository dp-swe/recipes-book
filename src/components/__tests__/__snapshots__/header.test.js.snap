// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should render correctly 1`] = `
<header
  className="header"
>
  <div
    className="header__title"
  >
    Recipes Book
  </div>
</header>
`;
